tracer/src/Datadog.Trace/DiagnosticListeners/DiagnosticManager.cs

```diff
@@ -3,23 +3,27 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-#if !NETFRAMEWORK
```

Comment:
Hmm I think this is introducing `System.Diagnostics.DiagnosticSource` as a reference which I don't think we should do.